tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/matter/matters.controller.js

```diff
@@ -148,7 +150,8 @@
                             headerCellTemplate: $templateCache.get('coldefheadertemplate.html'),
                             cellTemplate: cellTemplateContent,
                             enableCellEdit: true,
-                            position: value.position
+                            position: value.position,

```

Comment:
Added new properties called defaultVisibleInGrid to cater the needs of both MC and PC